shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/viewmodel/session/SessionDayViewModel.kt

```diff
@@ -26,6 +27,10 @@ class SessionDayViewModel(
             }
     )
 
+    var scrollState = ScrollState(firstVisibleItemIndex = 0, firstVisibleItemScrollOffset = 0)
```

Comment:
Scroll state is an implementation detail of view and shouldn't be in ViewModel. Let's look at it together tomorrow to find a better solution.